src/components/select/select.js

```diff
@@ -1136,11 +1142,11 @@ function SelectProvider($$interimElementProvider) {
            */
           function mouseOnScrollbar() {
             var clickOnScrollbar = false;
-            if (ev && (ev.currentTarget.children.length > 0)) {
+            if (mouseDownXPosition != null && ev && (ev.currentTarget.children.length > 0)) {
```

Comment:
I htought it would not be neccessary because there is always the mousedown before the mouseup, but maybe that is not the case when the mousedown was outside the mdSelect. I'll add that.